src/main/java/TaskManager.java

```diff
@@ -0,0 +1,77 @@
+public class TaskManager {
+    private Task[] tasks;
+    private int taskCount;
+    private static final String LINE_DIVIDER = "    ____________________________________________________________";
+    private static final String LIST_TASKS_PROMPT = "     Here are the tasks in your list:";
+    private static final String DONE_TASKS_PROMPT = "     Nice! I've marked this task as done:";
+    private static final String ADD_TASKS_PROMPT = "     Got it. I've added this task:";
+    private static final String ADD_TASKS_POST_PROMPT = "     Now you have %d tasks in the list.\n";
+    private static final String LIST_SINGLE_TASK_MESSAGE = "     %d.%s\n";
+    private static final String ADD_SINGLE_TASK_MESSAGE = "       %s\n";
+    private static final String DONE_SINGLE_TASK_MESSAGE = "       %s\n";
```

Comment:
Perhaps you can consider consolidating all the magic numbers/strings into a .class file